src/groundlight/client.py

```diff
@@ -303,3 +304,49 @@ def stop_inspection(self, inspection_id: str) -> str:
     def update_detector_confidence_threshold(self, detector_id: str, confidence_threshold: float) -> None:
         """Updates the confidence threshold of a detector given a detector_id."""
         self.api_client.update_detector_confidence_threshold(detector_id, confidence_threshold)
+
+
+def glcli():
```

Comment:
I think it might be better to put this in a separate file, for separation of concerns.